pkgs/development/python-modules/segyio/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, ninja
+, python
+, scikit-build
+, pytest
+, numpy
+}:
+
+stdenv.mkDerivation rec {
+  pname = "segyio";
+  version = "1.9.9";
+
+  postPatch = ''
+    # Removing unecessary build dependency
+    substituteInPlace python/setup.py --replace "'pytest-runner'," ""
+
+    # Fixing bug: This MINSIGSTKSZ is nowhere to be found inside the code base, so I removed this expression
+    substituteInPlace external/catch2/catch/catch.hpp --replace "32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ" 32768
+  '';
+
+  src = fetchFromGitHub {
+    owner = "equinor";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-L3u5BHS5tARS2aIiQbumADkuzw1Aw4Yuav8H8tRNYNg=";
+  };
+
+  nativeBuildInputs = [ cmake ninja python ];
+  buildInputs = [ scikit-build ];
```

Comment:
Most likely this should be nativeBuildInputs, too.

Reply:
I was in doubt but I agree